lib/common/widgets/pw_slider.dart

```diff
@@ -35,35 +35,147 @@ class _PwSliderState extends State<PwSlider> {
 
   @override
   Widget build(BuildContext context) {
+    final theme = Theme.of(context);
     return Column(
       mainAxisAlignment: MainAxisAlignment.start,
+      mainAxisSize: MainAxisSize.max,
       children: [
-        DetailsItem(
-          title: widget.title,
-          endChild: PwText(
-            '$_value',
-            style: widget.headerStyle,
-          ),
+        VerticalSpacer.largeX3(),
+        Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            PwText(
+              widget.title,
+              style: widget.headerStyle,
+              textAlign: TextAlign.start,
+            ),
+          ],
         ),
-        Padding(
-          padding: widget.padding ??
-              EdgeInsets.symmetric(
-                horizontal: Spacing.large,
-              ),
-          child: Slider(
-            min: widget.min,
-            max: widget.max,
-            value: _value,
-            divisions: 20,
-            onChanged: (value) {
-              setState(() {
-                _value = value;
-                widget.onValueChanged(_value);
-              });
-            },
-          ),
+        VerticalSpacer.largeX5(),
+        Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            PwText(
+              "${widget.min.toInt()}",
+              style: PwTextStyle.bodyBold,
+              textAlign: TextAlign.start,
+            ),
+            Expanded(
+              child: SliderTheme(
+                  data: SliderTheme.of(context).copyWith(
+                    showValueIndicator: ShowValueIndicator.always,
+                    trackHeight: Spacing.xSmall,
+                    trackShape: RoundedRectSliderTrackShape(),
+                    activeTrackColor: theme.colorScheme.secondary350,
+                    inactiveTrackColor: theme.colorScheme.neutral700,
+                    overlayColor: theme.colorScheme.neutralNeutral,
+                    thumbShape: _ThumbShape(),
+                    thumbColor: theme.colorScheme.secondary350,
+                    overlayShape: RoundSliderOverlayShape(overlayRadius: 5.0),
+                    tickMarkShape: RoundSliderTickMarkShape(),
+                    activeTickMarkColor: Colors.transparent,
+                    inactiveTickMarkColor: Colors.transparent,
+                    valueIndicatorColor: theme.colorScheme.secondary350,
+                    valueIndicatorTextStyle: theme.textTheme.footnote
+                        .copyWith(color: theme.colorScheme.neutral800),
+                  ),
+                  child: Slider(
+                    min: widget.min,
+                    max: widget.max,
+                    label: _value == 1.25
```

Comment:
Is there already a `const` somewhere for `1.25` that could be used?

Reply:
Yes!